benchmarks/bgp/decode_real_full_feed/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime/pprof"
+	"time"
+
+	"github.com/bio-routing/bio-rd/protocols/bgp/packet"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	nRuns = flag.Int("runs", 1, "# runs")
+)
+
+type task struct {
+	num int
+	raw *bytes.Buffer
+	msg *packet.BGPMessage
+}
+
+func main() {
```

Comment:
Let us use normal go benchmarks instead of own Programm. They support profiling too. https://medium.com/@felipedutratine/profile-your-benchmark-with-pprof-fb7070ee1a94

Reply:
My issue with that is, that they start measuring the whole function. I just want to measure certain actions on a populated data structure. If you know a way that integrates properly and allows this I'm open for change.

Reply:
we tried to refactor to benchmarks with a setup function but ran into issues with the buffers used. Since benchmarks are not part of the CI, we decided to no integrate it as a standard benchmark